src/main/scala/cromwell/engine/ExecutionTable.scala

```diff
@@ -0,0 +1,11 @@
+package cromwell.engine
+
+import cromwell.binding.{Call, WdlBinding}
+
+/**
+ * Corresponds to the "execution table" of our discussions.
+ */
+class ExecutionTable(binding: WdlBinding) {
+  /** TODO this is not adequate even for Sprint 2/3. */
+  def runnableCalls: Set[Call] = binding.workflow.calls
```

Comment:
Yes this is only adequate for my "hello world" workflow, it falls down even for three-step.  This is a TODO that needs doing soon.